Xamarin.Forms.Platform.UAP/CollectionView/StructuredItemsViewRenderer.cs

```diff
@@ -186,6 +186,25 @@ protected override void HandleLayoutPropertyChanged(PropertyChangedEventArgs pro
 						break;
 				}
 			}
+			else if (property.Is(StructuredItemsView.ItemsLayoutProperty))
```

Comment:
Changing the ItemsLayout used, switch between `FormsListView` and `FormsGridView` with the appropriate container in `ItemsViewRenderer.cs`, this block is not necessary.